smartconfig/file.py

```diff
@@ -62,7 +66,7 @@ def _update_mapping(
     return dest
 
 
-def load(path: Union[str, bytes, PathLike]) -> None:
+def load(path: Union[str, bytes, PathLike], *, yaml_loader: Type[yaml.Loader] = SmartconfigYamlFullLoader) -> None:
```

Comment:
Why is it a kw-only argument?

Reply:
I suspect that the API for this function may often change, I think it is better practice to put less used parameters as keyword-only.

Reply:
Not sure if I agree with this being frequently changing, but I don't really mind it being kw-only so, whatever.

Reply:
Undone by 2ecf31b.